adafruit_vl6180x.py

```diff
@@ -159,7 +159,7 @@ def read_lux(self, gain):
         elif gain == ALS_GAIN_20:
             lux /= 20
         elif gain == ALS_GAIN_40:
-            lux /= 20
+            lux /= 40
         lux *= 100
         lux /= 100  # integration time in ms
```

Comment:
This seems strange to me. Is that just to enfore float to be returned? Wouldnt then just

```suggestion
        lux /= 1
```

suffice?